data/core/units.cfg

```diff
@@ -247,7 +247,24 @@ The Dunefolk’s inquisitive and explorative nature does not preclude military s
         male_name= _ "race^Saurian"
         female_name= _ "race+female^Saurian"
         plural_name= _ "race^Saurians"
-        description= "" # wmllint: ignore
+        description= "Saurians are lizard-like creatures. Smaller and more slender than humans, they rarely stand taller than a ten year old child, though from tip of snout to end of tail a Saurian can be as long as the average man is tall. Light and nimble, the warriors prefer to fight as they hunt — slipping through enemy lines to target the weak and the injured while evading their attackers.
```

Comment:
This should be a translatable string (only saw this once it was already merged).

Reply:
Drat. Thanks.